### DISCUSSION TITLE
.env

### DISCUSSION THREAD

**Discussion by Init-Init** [foundry-rs/foundry#2113] [General]
i need help to configure an .env file for deploy a contract to rinkeby...
i have an error since 2 days: Failed to create wallet from private key: signature error
my .env file
export PRIVATE_KEY=xxxx
export RPC_URL=https://...
in my terminal 
forge script script/CryptoD.s.sol:cryptscript --rpc-url=$RPC_URL --private-key=$PRIVATE_KEY
Error: 
Failed to create wallet from private key: signature error

**Comment by onbjerg** [comment]
Are you sourcing your .env? Foundry will not do so automatically.

Also converting to a discussion because this does not seem like a bug :)

**Reply by Init-Init** [reply]
source does not work in windows!
there is an alternative?

**Reply by onbjerg** [reply]
You would have to use something like the Bash terminal bundled with Git, or WSL. Other than those two, I'm not sure, it's been a long time since I've used Windows.

**Reply by Init-Init** [reply]
thanks a lot git terminal work for me
